pages/index.vue

```diff
@@ -49,9 +50,13 @@
     </template>
 
     <div ref="whatson" class="container mt-4 text-white">
-      <h1 class="text-h1">What's On</h1>
+      <h1 class="text-h1 text-center md:text-left">What's On</h1>
```

Comment:
Did you mean to left-align this text? I suspect not! 😂 

<img width="1687" height="870" alt="Image" src="https://github.com/user-attachments/assets/fcc7bf62-1966-4a33-860e-1226cc7ea7eb" />